step-api/src/main/java/org/jenkinsci/plugins/workflow/util/StaplerReferer.java

```diff
@@ -38,8 +38,9 @@
 
 /**
  * JENKINS-19413 workaround.
+ * Will be deleted in the future.
  */
-class StaplerReferer {
+public class StaplerReferer {
```

Comment:
Cannot be `@Restricted` because I need to use it from sister plugins, unfortunately.